bin/fillTestData/Tribes.js

```diff
@@ -36,66 +55,90 @@ var tribeImageUUIDs = [
   '69a500a4-a16e-4c4d-9981-84fbe310d531'
 ];
 
-var addTribes = function (max) {
+var addTribes = function () {
   var index = 0;
+  var max = argv._[0];
+  var verbose = (argv.verbose === true);
+  var limit = (argv.limit === true);
 
   // Add tribes
   console.log('Generating ' + max + ' tribes...');
   if (max > 2000) {
     console.log('...this might really take a while... go grab some coffee!');
   }
 
-  console.log(chalk.white('--'));
-  console.log(chalk.green('Trustroots test tribes data'));
-  console.log(chalk.white('--'));
+  if (verbose) {
+    console.log(chalk.white('--'));
+    console.log(chalk.green('Trustroots test tribes data'));
+    console.log(chalk.white('--'));
+  }
+
+  // Override debug mode to use the option set by the user
+  config.db.debug = verbose;
```

Comment:
Would it work better to call the `--verbose` `--debug` instead?

Reply:
Thanks for all the feedback! I am totally fine with '--debug'. I just wanted to make sure it was something that expressed 'loud' since 'quiet' is now the default.